src/zorak/cogs/admin/verification_on_verified.py

```diff
@@ -19,9 +22,43 @@ def __init__(self, bot):
     def is_verified(self, member):
         if "✅" in [role.name for role in member.roles]:
             return True
+        
+    async def send_wrong_button_message(self, guild, member):
+        button_message = f"""
+            Hi there, {member.mention}
+            you have pressed the wrong button to verify yourself. 
+
+            Make sure you press the **GREEN** button to verify yourself.
+            Please join the server again and try again.
+            """
+        # Send Welcome Message
+        try:
+            await member.send(button_message)
+        except discord.errors.Forbidden as catch_dat_forbidden:
+            logger.debug(f'{member.name} cannot be sent a DM.')
```

Comment:
Doesn't give many options to users who have DM's off, which are often more mature users. Or bots, I have no idea.